assets/css/blueprint.css

```diff
@@ -4270,7178 +4272,7307 @@ a.has-text-danger:focus {
-    top: 53px;
-    left: 29px;
+  animation-delay: -0.9s;
+  top: 53px;
+  left: 29px;
 }
 .lds-default div:nth-child(11) {
-    animation-delay: -1s;
-    top: 50px;
-    left: 41px;
+  animation-delay: -1s;
+  top: 50px;
+  left: 41px;
 }
 .lds-default div:nth-child(12) {
-    animation-delay: -1.1s;
-    top: 41px;
-    left: 50px;
+  animation-delay: -1.1s;
+  top: 41px;
+  left: 50px;
 }
 
 @keyframes lds-default {
-    0%,
-    20%,
-    80%,
-    100% {
-        transform: scale(1);
-    }
-    50% {
-        transform: scale(1.5);
-    }
+  0%,
+  20%,
+  80%,
+  100% {
+    transform: scale(1);
+  }
+  50% {
+    transform: scale(1.5);
+  }
 }
 
 .alias {
-    cursor: alias;
+  cursor: alias;
 }
 .all-scroll {
-    cursor: all-scroll;
+  cursor: all-scroll;
 }
 .auto {
-    cursor: auto;
+  cursor: auto;
 }
 .cell {
-    cursor: cell;
+  cursor: cell;
 }
 .context-menu {
-    cursor: context-menu;
+  cursor: context-menu;
 }
 .col-resize {
-    cursor: col-resize;
+  cursor: col-resize;
 }
 .copy {
-    cursor: copy;
+  cursor: copy;
 }
 .crosshair {
-    cursor: crosshair;
+  cursor: crosshair;
 }
 .default {
-    cursor: default;
+  cursor: default;
 }
 .e-resize {
-    cursor: e-resize;
+  cursor: e-resize;
 }
 .ew-resize {
-    cursor: ew-resize;
+  cursor: ew-resize;
 }
 .grab {
-    cursor: -webkit-grab;
-    cursor: grab;
+  cursor: -webkit-grab;
+  cursor: grab;
 }
 .grabbing {
-    cursor: -webkit-grabbing;
-    cursor: grabbing;
+  cursor: -webkit-grabbing;
+  cursor: grabbing;
 }
 .help {
-    cursor: help;
+  cursor: help;
 }
 .move {
-    cursor: move;
+  cursor: move;
 }
 .n-resize {
-    cursor: n-resize;
+  cursor: n-resize;
 }
 .ne-resize {
-    cursor: ne-resize;
+  cursor: ne-resize;
 }
 .nesw-resize {
-    cursor: nesw-resize;
+  cursor: nesw-resize;
 }
 .ns-resize {
-    cursor: ns-resize;
+  cursor: ns-resize;
 }
 .nw-resize {
-    cursor: nw-resize;
+  cursor: nw-resize;
 }
 .nwse-resize {
-    cursor: nwse-resize;
+  cursor: nwse-resize;
 }
 .no-drop {
-    cursor: no-drop;
+  cursor: no-drop;
 }
 .none {
-    cursor: none;
+  cursor: none;
 }
 .not-allowed {
-    cursor: not-allowed;
+  cursor: not-allowed;
 }
 .pointer {
-    cursor: pointer;
+  cursor: pointer;
 }
 .progress {
-    cursor: progress;
+  cursor: progress;
 }
 .row-resize {
-    cursor: row-resize;
+  cursor: row-resize;
 }
 .s-resize {
-    cursor: s-resize;
+  cursor: s-resize;
 }
 .se-resize {
-    cursor: se-resize;
+  cursor: se-resize;
 }
 .sw-resize {
-    cursor: sw-resize;
+  cursor: sw-resize;
 }
 .text {
-    cursor: text;
+  cursor: text;
 }
 .url {
-    cursor: url(myBall.cur), auto;
+  cursor: url(myBall.cur), auto;
 }
 .w-resize {
-    cursor: w-resize;
+  cursor: w-resize;
 }
 .wait {
-    cursor: wait;
+  cursor: wait;
 }
 .zoom-in {
-    cursor: zoom-in;
+  cursor: zoom-in;
 }
 .zoom-out {
-    cursor: zoom-out;
+  cursor: zoom-out;
 }
 
 /*# sourceMappingURL=blueprint.css.map */
+
+.min-height-mobile {
+  min-height: 398px;
```

Comment:
Wanted a sanity check here: Where does this `398` value come from? I couldn't find it defined in the Figma for the website template.

Reply:
where did you see the figma ah? i wasn't aware there was one ._. how i found this was using the _existing_ template (actual amp/netlify site), i changed to mobile layout and found the `398`

Reply:
cleared with design, no actual figma for this lmao